core/sql/regress/tools/runregr_udr.ksh

```diff
@@ -851,9 +851,9 @@ if [ $LINUX -ne 0 ]; then
 fi
 
```

Comment:
There is a lot of cleanup that could be done here, but that would require more testing and take more time. Is it ok if we keep this PR to just the typo?